test/neon-animated-pages-descendant-selection.html

```diff
@@ -57,8 +58,11 @@
 
 <dom-module id="test-element">
   <template>
-    <neon-animated-pages entry-animation="test-animation" animate-initial-selection>
-      <content></content>
+    <neon-animated-pages
+        entry-animation="test-animation"
+        attr-for-selected="index"
+        animate-initial-selection>
+      <slot></slot>
```

Comment:
We should use `WebComponentsReady` instead of `HTMLImports.whenReady` in the script below, or just declare this `test-element` in a separate file and import it here